resolwe/test_helpers/test_runner.py

```diff
@@ -218,6 +218,7 @@ def _create_test_dirs():
     paths = _sequence_paths([resolwe_settings.FLOW_EXECUTOR_SETTINGS[i] for i in items])
     for item, path in zip(items, paths):
         resolwe_settings.FLOW_EXECUTOR_SETTINGS[item] = path
+    return [resolwe_settings.FLOW_EXECUTOR_SETTINGS[item] for item in items]
```

Comment:
Why are you iterating over items twice instead of just returning `paths`?

Reply:
Err...